server/scripts/fetchdata/CardImport.js

```diff
@@ -0,0 +1,63 @@
+/*eslint no-console:0 */
+
+const fs = require('fs');
+const mkdirp = require('mkdirp');
+const path = require('path');
+
+const CardService = require('../../services/CardService.js');
+
+class CardImport {
+    constructor(db, dataSource, imageSource, imageDir) {
+        this.db = db;
+        this.dataSource = dataSource;
+        this.imageSource = imageSource;
+        this.imageDir = imageDir;
+        this.cardService = new CardService(db);
+    }
+
+    async import() {
+        try {
+            await Promise.all([this.importCards(), this.importPacks()]);
```

Comment:
why not just `await this.importCards();\nawait this.importPacks(); `?

Reply:
The `all` allows them to be fetched in parallel, which was the previous behavior. I could split it into two `await`s for clarities sake if you want, but that will change the behavior to fetch each piece of data sequentially.

Reply:
Ahha, of course, no it's find as is